### DISCUSSION TITLE
庆祝一下公司的俄罗斯高管生日

### DISCUSSION THREAD

**Discussion by AnkaAlioth** [share121/inter-knot#2777] [其他]
<img width="621" alt="52b6ecd9-8731-41ba-b157-bd6015963456" src="https://github.com/user-attachments/assets/5b444127-5c16-4553-a7aa-9b9cc3a3405c">

以及偷听到公司里爱沙尼亚女秘书的对话
<img src="https://img-static.mihoyo.com/communityweb/upload/ca04d0b86f7073fe2f940c7620cf6e6a.png" alt="偷看" width="75" >

**Comment by demolute** [comment]
我就说这公司有黑幕吧，这个秘书怎么上位的背后绝对有鬼
